book/chapters/chapter13/beyond_regression_and_classification.qmd

```diff
@@ -128,12 +128,14 @@ By using `po("learner_cv")` for internal resampling and `po("tunethreshold")` to
 ## Survival Analysis {#sec-survival}
 
 `r index("Survival analysis")` is a field of statistics concerned with trying to predict/estimate the time until an event takes place.
-This predictive problem is unique as survival models are trained and tested on data that may include 'censoring', which occurs when the event of interest does *not* take place.
+This predictive problem is unique because survival models are trained and tested on data that may include censoring, which occurs when the exact event time is not observed.
+The most common type of censoring is 'right censoring', which happens when the event of interest has not yet occurred by the time observation ends.
+For the rest of this section, censoring means right censoring unless otherwise stated.
 Survival analysis can be hard to explain in the abstract, so as a working example consider a marathon runner in a race.
 Here the 'survival problem' is trying to predict the time when the marathon runner finishes the race.
-However, if the event of interest does not take place (e.g., the marathon runner gives up and does not finish the race), they are said to be censored.
+However, if the organizers stop recording finish times after a certain point, then any runner still running beyond that time will be censored.
 Instead of throwing away information about censored events, survival analysis datasets include a status variable that provides information about the 'status' of an observation.
```

Comment:
```suggestion
Instead of discarding such incomplete observations, survival analysis incorporates a status variable to reflect whether the event was observed.
```